app/views/shared/_document_collections_email_signup.html.erb

```diff
@@ -14,5 +14,5 @@
   <%= render 'shared/single_page_notification_button', {
     content_item: @content_item,
     skip_account: @has_govuk_account ? "false" : "true"
-    } %>
+    } if @content_item.display_single_page_notification_button? %>
```

Comment:
I suppose this `if ...` can be put in place of `else` in line 13 (as `elsif`) which I think would make it faster to read?

```
<% elsif @content_item.display_single_page_notification_button? %>
```